src/wp-content/themes/twentytwentyone/style.css

```diff
@@ -5740,11 +5749,13 @@ h1.page-title {
 }
 
 .has-black-background-color[class] {
+	--wp--preset--color--black: var(--global--color-black);
 	background-color: var(--global--color-black);
 }
 
 .has-black-background-color[class] > [class*=__inner-container] {
 	--local--color-background: var(--global--color-black, #000);
+	--wp--preset--color--black: var(--local--color-background);
```

Comment:
It seems strange to be setting preset color black to --local--color-background. It seems more natural for a background to reference a preset than a preset referencing a background. Couldn't we set the presents globally all the in the same place:
```
:root {
--wp--preset--color--black: #cccc, 
--wp--preset--color--white: #cccc, 
....
}
```
And then never set additional presets?

Reply:
Probably. I didn't have the time to investigate how this theme is built deeply. I wanted this PR's scope to remain small, focused in fixing the issue.